mlblocks_primitives/mlprimitives.timeseries_errors.extract_anomalies.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "mlprimitives.timeseries_errors.extract_anomalies",
+    "author": "Ihssan Tinawi <itinawi@mit.edu>",
+    "description": "mlprimitives.timeseries_errors.extract_anomalies",
+    "classifiers": {
+        "type": "preprocessor",
+        "subtype": "feature_extractor"
+    },
+    "modalities": ["timeseries"],
+    "primitive": "mlprimitives.timeseries_errors.extract_anomalies",
+    "produce": {
+        "args": [
+            {
+                "name": "y_true",
+                "type": "np.array"
+            },
+            {
+                "name": "smoothed_errors",
```

Comment:
This should be a hyperparameter

Reply:
Why should it be? It is the output of `get_forecast_error` once it takes in `y_true` and `y_hat` and contains the sum of squared errors. It's not something that we can tune.